### PR TITLE
Add `vote.clucoin.com` to domains

### PR DESCRIPTION
**Pull request by M4cs**
Adds `vote.clucoin.com` subdomain

#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [X] Add custom domain for your space
- [ ] Add alias for migrate your space
- ~Add images for your space~ (deprecated) Add the image URL on the space settings directly
- ~Edit your space with GitHub?~ (deprecated) Please migrate your space to ENS https://docs.snapshot.page/spaces/migrate
- ~Create your space with GitHub~ (deprecated) Create your space with ENS instead https://docs.snapshot.page/spaces/create

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "gov.hyruleswap.com": "hyruleswap.eth",
   "complus.network": "complus.eth",
   "vote.axion.network": "axiondao.eth",
+  "vote.clucoin.com": "macbs.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:26]: Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
  "vote.clucoin.com": "macsb.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 1 above)
Comment (M4cs) [reply spaces/domains.json:26]: Somebody had purchased clu.eth and clucoin.eth before we decided to get an ENS address. Are you saying my ENS address must relate to `clucoin` as well? I own `getclucoin.eth` as well, would that ENS suffice?

Comment (M4cs) [reply spaces/domains.json:26]: Sorry I see what you are saying! I typoed

Comment (M4cs) [reply spaces/domains.json]: Should be fixed now

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**